src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1010,8 +1011,16 @@ public void srandmember(final byte[] key, final int count) {
     sendCommand(SRANDMEMBER, key, toByteArray(count));
   }
 
-  public void clientKill(final byte[] client) {
-    sendCommand(CLIENT, Keyword.KILL.raw, client);
+  public void clientKill(final byte[] ipPort) {
```

Comment:
does it still make sense to keep this format?. As this will be merged in 3.0 we can completely remove all the `ip` and `port` combinations and just use the `ClientKillParams` one. WDYT?

Reply:
Redis supports both form. So we should keep both.
https://redis.io/commands/client-kill